lang/cy.json

```diff
@@ -545,45 +502,36 @@
     "trackLpaProgress": "Welsh",
     "backToDashboard": "Welsh",
     "myLpa": "Welsh {{ .LpaType }} welsh",
-
     "yourLegalRightsAndResponsibilities": "Welsh",
-    "yourLegalRightsAndResponsibilitiesContent": "<p class=\"govuk-body\">Welsh</p>",
-
+    "yourLegalRightsAndResponsibilitiesContent": "<p class=\"govuk-body-l\">Cyn llofnodi, rhaid i chi ddarllen eich hawliau a'ch cyfrifoldebau cyfreithiol.</p><h2 class=\"govuk-heading-m\">Sut y dylai eich atwrneiod weithredu</h2><p class=\"govuk-body\">Drwy lofnodi'ch LPA, rydych yn penodi eich atwrneiod i wneud penderfyniadau ar eich rhan.</p> <p class=\"govuk-body\">Rhaid i'ch atwrneiod ddilyn <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/mental-capacity-act-code-of-practice\">Cod Ymarfer y Ddeddf Galluedd Meddyliol</a>:</p><ul class=\"govuk-list govuk-list--bullet\"><li>Rhaid iddynt gymryd yn ganiataol y gallwch wneud eich penderfyniadau eich hun, oni bai y sefydlir na allwch wneud hynny </li><li>Rhaid iddynt eich helpu i wneud cymaint o'ch penderfyniadau eich hun ag y gallwch.</li><li>Rhaid iddynt gymryd pob cam ymarferol i'ch helpu i wneud penderfyniad. Rhaid iddynt ond eich trin fel rhywun na allant wneud penderfyniad os nad ydynt wedi llwyddo i'ch helpu i wneud penderfyniad drwy'r camau hynny.</li><li>Rhaid iddynt beidio â'ch trin fel rhywun na allant wneud penderfyniad oherwydd eich bod wedi gwneud penderfyniad annoeth. .</li><li>Rhaid iddynt weithredu a gwneud penderfyniadau er eich lles gorau pan na fyddwch yn gallu gwneud penderfyniad.</li><li>Cyn iddynt wneud penderfyniad neu weithredu ar eich rhan, rhaid iddynt ystyried unrhyw opsiwn sy'n llai cyfyngu ar eich hawliau a'ch rhyddid a allai sicrhau'r un canlyniad.</li></ul><h2 class=\"govuk-heading-m\">Sut y gellir defnyddio'ch LPA</h2><ul class=\"govuk-list govuk-list--bullet\"><li>Pan fyddwch yn llofnodi'ch LPA, ynghyd â'ch holl atwrneiod, amnewid atwrneiod a darparwr tystysgrif, rydych yn ffurfio cytundeb cyfreithiol rhyngoch chi (gweithred).</li><li>Dim ond os yw wedi'i chofrestru â Swyddfa'r Gwarcheidwad Cyhoeddus (OPG) y gellir defnyddio'ch LPA.</li><li> Gallwch ganslo eich LPA unrhyw bryd os oes gennych alluedd meddyliol. <a class=\"govuk-link\" href=\"https://www.gov.uk/power-of-attorney/end\">Dysgu rhagor am sut i ganslo eich LPA</a>.</li><li>Ni all eich atwrneiod ddefnyddio'ch LPA i wneud newidiadau i'ch ewyllys.</li><li>Bydd eich LPA yn dod i ben pan fyddwch yn marw.</li><li> X1954X]</ul>",
```

Comment:
the very end of this change starts a new list item without closing it

```html
<li> X1954X]</ul>"
```

which if we want to keep it (it looks like junk to me) should be 

```html
<li> X1954X]</li></ul>"
```